contracts/marketplace/src/multitest.rs

```diff
@@ -1293,7 +1294,7 @@ mod tests {
         let marketplace_id = router.store_code(contract_nft_marketplace());
         let msg = crate::msg::InstantiateMsg {
             operators: vec!["operator".to_string()],
-            trading_fee_percent: TRADING_FEE_PERCENT,
+            trading_fee: TRADING_FEE,
```

Comment:
Should we rename this to `trading_fee_basis_points` to make it more clear?

Reply:
Yes, excellent idea.